config/checker/javax.swing.astub

```diff
@@ -41,6 +41,12 @@ class JTable {
   void initializeLocalVars(@UnknownInitialization JTable this);
 }
 
+class SwingUtilities {
+  @SafeEffect
+  static boolean isEventDispatchThread();
```

Comment:
Just realized today that shouldn't block calling this method from outside UI thread